gateway/api/connections/queries_schema.go

```diff
@@ -8,6 +8,11 @@ import (
 
 // getTablesQuery returns the query to list only the tables of a database
 func getTablesQuery(connType pb.ConnectionType, dbName string) string {
+	// Check for DynamoDB (custom type with dynamodb subtype)
+	if string(connType) == "custom" && connType.String() == "custom/dynamodb" {
+		return getDynamoDBTablesQuery()
+	}
```

Comment:
This statement seems odd. The `connType` cannot be both things. You're also validating if the script is empty after calling this function

Reply:
Thx! I also added to pb.ConnectionType of the DynamoDB to keep everything using the same source.